dev-resources/docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,88 @@
+volumes:
```

Comment:
we literally put this in its own repo. What is it doing back here?

Reply:
not addressed

Reply:
I thought it made more sense as part of elrr.

Reply:
this is datasync, that doesn't make sense. It's got its own repo.